### DISCUSSION TITLE
Does Directus 9 support mysql8's default authentication (caching_sha2_password)

### DISCUSSION THREAD

**Discussion by bobbyphilip** [directus/directus#14732] [Archive]
I see an error on startup
Error: ER_NOT_SUPPORTED_AUTH_MODE: Client does not support authentication protocol requested by server; consider upgrading MySQL client
I had to change the default-authentication-plugin for mysql to get this to work

**Comment by azrikahar** [comment]
The readme does mention this:

> MySQL 5.7.8+ / 8+ (with [mysql_native_password](https://dev.mysql.com/doc/refman/8.0/en/upgrading-from-previous-series.html#upgrade-caching-sha2-password-compatible-connectors))

as well as the self-hosted installation docs: https://docs.directus.io/self-hosted/installation/cli/#_1-confirm-minimum-requirements (footnote number 1)

That being said, this is technically something to do with the `mysql` driver in Knex (they are tracking this in https://github.com/mysqljs/mysql/issues/2499). `mysql2` does seem to support MySQL 8's default auth, but that may require more research than just "switching" to it.
